lib/SPIRV/SPIRVToOCL.cpp

```diff
@@ -498,6 +502,33 @@ void SPIRVToOCL::visitCallSPIRVImageMediaBlockBuiltin(CallInst *CI, Op OC) {
       &Attrs);
 }
 
+void SPIRVToOCL::visitCallSPIRVCvtBuiltin(CallInst *CI, Op OC,
+                                          StringRef DemangledName) {
+  AttributeList Attrs = CI->getCalledFunction()->getAttributes();
+  mutateCallInstOCL(
+      M, CI,
+      [=](CallInst *Call, std::vector<Value *> &Args) {
+        std::string CastBuiltInName;
+        if (isCvtFromUnsignedOpCode(OC))
+          CastBuiltInName = "u";
+        CastBuiltInName += kOCLBuiltinName::ConvertPrefix;
+        Type *DstTy = Call->getType();
+        CastBuiltInName +=
+            mapLLVMTypeToOCLType(DstTy, !isCvtToUnsignedOpCode(OC));
+        if (DemangledName.find("_sat") != StringRef::npos || isSatCvtOpCode(OC))
+          CastBuiltInName += "_sat";
+        auto Loc = DemangledName.find("_rt");
+        Value *Src = Call->getOperand(0);
+        assert(Src && "Invalid SPIRV convert builtin call");
+        Type *SrcTy = Src->getType();
```

Comment:
```suggestion
        Value *Src = Call->getOperand(0);
        assert(Src && "Invalid SPIRV convert builtin call");
        Type *SrcTy = Src->getType();
        auto Loc = DemangledName.find("_rt");
```

To make `Loc` declaration closer to its first usage